content/blog/2022/10/2022-10-14-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,172 @@
+(also a sponsor) metric collection on ephemeral agents. We hope to also
+use traces in the future
+* New “All-in-one” agent template built with Packer: whether your builds
+are running on a VM or a container, in AWS, Azure or Digital Ocean, you
+have the same tools in the same locations
+
+== Platform Update
+
+* Loaner computers for ppc64le are gone
+** IBM suspended the loaner machine program
+* arm/v7 agent Docker images
+* Linux packages use systemd instead of System V init
+* Exit lifecycle change in the Docker images
+* Container image deprecation for the blueocean container
+(jenkinsci:blueocean)
+* Container repository management for Jenkins agents
+** Simplifying the release process
+* JDK 11 now required since Jenkins 2.357 and 2.361.1 LTS
+** Ongoing minor tasks
+*** Dropping Java 8 from the Docker agents - merged, not yet released
+*** Plugins beginning to require Java 11, ongoing process
+** Jenkins works with Java 17 since 2.355 and 2.346.1 LTS
+** Ongoing, a few recent reports, working well
+** Jenkins infra planning to put one or more of the infra machines on JDK
+17
+** Timeline indicates end of Java 11 first level maintenance Sep 2023,
+security fixes for several more years after end of first level
+maintenance
+** More than 11 000 Java 17 installs reporting worldwide with Jenkins in
+September
+
+== Documentation Update
+
+=== DigitalOcean and Jenkins partnership continues to grow!
+
+Thanks to DigitalOcean’s generosity, Jenkins has access to more resources than ever before, such as utilizing the DigitalOcean platform to continue development and share progress made. We can dedicate our time and resources more fully, allowing users to engage and make an impact. Read the story https://www.jenkins.io/blog/2022/09/19/digital-ocean-sponsorship/[+++HERE+++].
+
+=== image:/images/post-images/2022-10-14-jenkins-newsletter/image6.png[image,width=247,height=139]
+
+===  Java 11 or Newer
+
+Beginning with Jenkins 2.357 (released on June 28, 2022) and the
+forthcoming 2.361.1 LTS release, Jenkins requires Java 11 or newer.
+Additionally, beginning with Jenkins 2.355 (released on June 14, 2022)
+and Jenkins 2.346.1 LTS (released on June 22, 2022), Jenkins supports
+Java 17. Plugins have already been prepared in
+https://issues.jenkins.io/browse/JENKINS-68446[JENKINS-68446]. Use the
+Plugin Manager to upgrade all plugins before _and_ after upgrading to
+Jenkins 2.357. Read the post
+https://www.jenkins.io/blog/2022/06/28/require-java-11/[+++HERE+++].
+
+=== image:/images/post-images/2022-10-14-jenkins-newsletter/image3.png[image,width=239,height=139]
+
+=== Thinking About Contributing to Open Source?
+
+Ponder no more! We’ve created the
+https://www.jenkins.io/doc/developer/tutorial-improve/["Improve a
+plugin"] developer tutorial for new contributors. We’d love to work with
+you!
+
+=== About Blue Ocean
+
+image:/images/post-images/2022-10-14-jenkins-newsletter/image4.png[image,width=624,height=286]
```

Comment:
I would reproduce the text here instead of a text screenshot.

Reply:
Done, thanks.